### PR TITLE
linux setup file

### PR DESCRIPTION
**Pull request by qingyggg**
Optimize setup file

### CODE REVIEW

File: deploy/scripts/README.md [hunk 1]
```diff
@@ -10,14 +10,7 @@
...
 
-# setup k8s cluster
-sh install-k8s.sh
-
-# setup laf core
-sh install-laf-core.sh
-
-# apply laf cluster resources
-kubectl apply -f init-laf/
+sh bootstrap-linux.sh
```
Comment (maslow) [inline deploy/scripts/README.md:13]: Where is `bootstrap-linux.sh`

Comment (qingyggg) [reply deploy/scripts/README.md:13]: sorry,I forgot to add it,now ,It added

**Comment by qingyggg** [comment]
now its ok,I push the file

**Comment by qingyggg** [comment]
1.I am a linux user,my computer host os is linux
2.the reason of I pr this is make linux user setup more comfortable
3.for example,if you are debian/rehat distro user,you use this can install sealos,but others cannot,like arch,suse,so I add install sealos from the source step

**Comment by qingyggg** [comment]
When I sh install-k8s.sh,the progress always apt update and waste lots of time,so I will add sealos version check and run sealos reset command,they will not be excute unless sealos is exist.

what do you think of this adeal?

**Comment by qingyggg** [comment]
and I think,sealos package can add to brew,its a package manager,it can be used in all linux distros,now apt and yum are only serve ubuntu/debian and centos/redhat

File: deploy/scripts/install-k8s.sh [hunk 2]
```diff
@@ -32,6 +32,14 @@ else
   echo "yum not installed"
 fi
 
+#arch,suse distro condition
+if [ -x "$(command -v sealos)"];then
+  wget https://github.com/labring/sealos/releases/download/v4.1.3/sealos_4.1.3_linux_amd64.tar.gz \
+   && tar zxvf sealos_4.1.3_linux_amd64.tar.gz sealos && chmod +x sealos && mv sealos /usr/bin
```
Comment (maslow) [inline deploy/scripts/install-k8s.sh:38]: Does this script seem to work only in AMD64?

**Review by maslow — CHANGES_REQUESTED** [review]

File: deploy/scripts/install-k8s.sh [hunk 3]
```diff
@@ -32,6 +32,14 @@ else
   echo "yum not installed"
 fi
 
+#arch,suse distro condition
```
[the remark this answers is not in the document]
Comment (qingyggg) [reply deploy/scripts/install-k8s.sh:35]: add sealos installation support to other linux distribution

File: deploy/scripts/install-k8s.sh (hunk 2 above)
Comment (qingyggg) [reply deploy/scripts/install-k8s.sh:38]: okay,I will improve later

Comment (qingyggg) [reply deploy/scripts/install-k8s.sh:40]: I add arm support now!!!

**Pull request closed without merging**